client/src/pages/Prospects.js

```diff
@@ -56,6 +61,30 @@ const useStyles = makeStyles((theme) => ({
 const Prospects = () => {
   const classes = useStyles();
 
+  const prepareData = () => {
+    const results = [];
+
+    //replace data with real data
+    const data = SampleData();
+    const cloudIcon = <CloudIcon className="fas fa-cloud" style={{color: "grey"}} />
+    data.map(each => {
+      const obj = {
+        'check': 'check',
+        'Email': each.email,
+        cloudIcon,
+        'Status': 'working',
+        'Owner': each.owner,
+        'Campaigns': each.campaigns,
+        'Last Contacted': each.lastContacted,
+        'Emails...': each.emails
```

Comment:
Currently, the Prospects Model only has:
    id, email, name, status, owner_id, tags 

Not much of a problem right now, and should be a quick fix in the future. 
Just wanted to point that out there